app/src/organisms/ChooseRobotSlideout/index.tsx

```diff
@@ -316,7 +331,163 @@ export function ChooseRobotSlideout(
     </Flex>
   )
 
-  const pageTwoBody = <Flex>TODO</Flex>
+  const runtimeParams = runTimeParametersOverrides.map(
+    (runtimeParam, index) => {
+      if ('choices' in runtimeParam) {
+        const dropdownOptions = runtimeParam.choices.map(choice => {
+          return { name: choice.displayName, value: choice.value }
+        }) as DropdownOption[]
+        return (
+          <DropdownMenu
+            key={runtimeParam.variableName}
+            filterOptions={dropdownOptions}
+            currentOption={
+              dropdownOptions.find(choice => {
+                return choice.value === runtimeParam.value
+              }) ?? dropdownOptions[0]
+            }
+            onClick={choice => {
+              const clone = runTimeParametersOverrides.map((parameter, i) => {
+                if (i === index) {
+                  return {
+                    ...parameter,
+                    value:
+                      dropdownOptions.find(option => option.value === choice)
+                        ?.value ?? parameter.default,
+                  }
+                }
+                return parameter
+              })
+              setRunTimeParametersOverrides(clone)
+            }}
+            title={runtimeParam.displayName}
+            caption={runtimeParam.description}
+            width="100%"
+            dropdownType="neutral"
+          />
+        )
+      } else if (runtimeParam.type === 'int' || runtimeParam.type === 'float') {
+        return (
+          <InputField
+            key={runtimeParam.variableName}
+            type="number"
+            units={runtimeParam.suffix}
+            placeholder={runtimeParam.value as string}
```

Comment:
since the run time param here is a number since its type int or float, rather than type casting it as a string, can we do this?

```suggestion
            placeholder={runtimeParam.value.toString()}
```

if that doesn't work then above line 370, we can make a `const value = runtimeParam.value as number` And then use that for the placeholder and the value.